test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Batch/Client/DefaultBatchGeneratedClient.cs

```diff
@@ -1,4 +1,11 @@
-﻿//------------------------------------------------------------------------------
+//-----------------------------------------------------------------------------
+// <copyright file="DefaultBatchGeneratedClient.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+//------------------------------------------------------------------------------
```

Comment:
Verify if this file is auto-generated. If yes, do we need to add the copyright header?

Reply:
@marabooy You can ignore this comment. I think it's right for this file to have the copyright header. It just happens that it was auto-generated somewhere else before being added to this project